cognite/client/data_classes/_base.py

```diff
@@ -527,3 +532,93 @@ def dump(self, camel_case: bool = False) -> Dict[str, Any]:
 
 
 T_CogniteFilter = TypeVar("T_CogniteFilter", bound=CogniteFilter)
+
+
+class EnumProperty(Enum):
+    def as_reference(self) -> list[str]:
```

Comment:
Elegant, and I like it. But it is slightly magical, so I prefer having it as is, but I can be convinced otherwise as a good argument for doing this change is that it is more bug proof.